runelite-mixins/src/main/java/net/runelite/mixins/RSModelMixin.java

```diff
@@ -276,4 +280,25 @@ public Polygon getConvexHull(int localX, int localY, int orientation)
 
 		return p;
 	}
+
+	@Override
+	@Inject
+	public void drawOutline(int localX, int localY, int localZ, int orientation,
```

Comment:
I would probably move entire Rasterizer to runelite-client, and just expose the model properties and then just have some method overrides for actors, objects etc. Mainly because the ExecutorService running in API is quite unpleasant, also API is probably not good place for such amount of logic Rasterizer has.

Then we would be able to use our ExecutorService that we use in client, and maybe even make the triggering of the entire reset procedure slightly smarter (maybe use Guava caches etc)